README.md

```diff
@@ -5,9 +5,9 @@ Project 2: Building with Responsive Design
 
 Project 2 of Microverse 2.0 program: Building with Responsive Design
 
-We were asked to make a clone of The New World page using media query to work with 3 screen sizes: mobile, tablet, and desktop using mediaqueries.
+We were asked to make a clone of The Next Web page using media query to work with 3 screen sizes: mobile, tablet, and desktop using mediaqueries.
 
-![READMESNEAKPEAK](https://user-images.githubusercontent.com/56790126/108226304-75532f00-7145-11eb-9f71-7916903fbc67.png)
```

Comment:
- [ ] On the readme, please change the term `Project Name` to the actual name of the project.